build.gradle.kts

```diff
@@ -68,23 +68,23 @@ protobuf {
   }
 }
 
-val protoArchive = file("$buildDir/archives/opentelemetry-proto-$protoVersion.zip")
+val protoArchive = layout.buildDirectory.file("archives/opentelemetry-proto-$protoVersion.zip")
 
 tasks {
   test {
     useJUnitPlatform()
   }
 
   val downloadProtoArchive by registering(Download::class) {
-    onlyIf { !protoArchive.exists() }
```

Comment:
I assume that you removed this because gradle is smart enough to figure out that it doesn't need to download it again when the destination file already exists